server/sonar-db-dao/src/main/java/org/sonar/db/purge/PurgeConfiguration.java

```diff
@@ -50,19 +52,19 @@ public PurgeConfiguration(IdUuidPair rootProjectId, String[] scopesWithoutHistor
   }
 
   public static PurgeConfiguration newDefaultPurgeConfiguration(Configuration config, IdUuidPair idUuidPair, Collection<String> disabledComponentUuids) {
-    String[] scopes = new String[] {Scopes.FILE};
+    Collection<String> qualifiersWithoutHistoricalData = asList(Qualifiers.FILE, Qualifiers.UNIT_TEST_FILE);
```

Comment:
changes to this method must be rollbacked to define scopes which are actually scopes and not qualifiers in the default PurgeConfiguration